citus-enterprise.spec

```diff
@@ -54,6 +54,9 @@ if [ "$(printf '%s\n' "$requiredgccver" "$currentgccver" | sort -V | tail -n1)"
     fi
 fi
 
+# also rename safestringlib license file to prevent naming collisions
+mv vendor/safestringlib/LICENSE vendor/safestringlib/safestringlib-LICENSE
```

Comment:
```suggestion
mv vendor/safestringlib/LICENSE safestringlib-LICENSE
```

Reply:
nit, Should we use `cp` instead, and keep the original license file in place?